src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -0,0 +1,211 @@
+
+Service workers, which are widely supported in current browsers, offer an alternative to the offline experience provided by AppCache.
+
+We've provided a [polyfill library](https://github.com/GoogleChromeLabs/sw-appcache-behavior) that uses a service worker to replicate some of the functionality of AppCache, though it does not replicate the entire AppCache interface. In particular, it does not provide a replacement for the `window.applicationCache` interface or the related AppCache events.
+
+For more complex cases, libraries like [Workbox](https://developers.google.com/web/tools/workbox/) provide an easy way to create a modern service worker for your web app.
+
+### Service workers and AppCache are mutually exclusive
+
+While working on your migration strategy, please keep in mind that Chrome will disable AppCache functionality on any page that's loaded under the [control](https://developers.google.com/web/fundamentals/primers/service-workers/lifecycle#scope_and_control) of a service worker. In other words, as soon as you deploy a service worker that controls a given page, you're no longer able to use AppCache on that page.
+
+Because of this, we recommend that you do not attempt to migrate to service workers piece-by-piece. It would be a mistake to deploy a service worker that only contains some of your caching logic. You can not fall back on AppCache to "fill in the gaps."
+
+Similarly, if you deploy a service worker prior to AppCache removal, and then discover that you need to roll back to your previous AppCache implementation, you need to ensure that you [unregister](https://stackoverflow.com/a/33705250/385997) that service worker. As long as there's a registered service worker in scope for a given page, AppCache will not be used.
+
+## The cross-platform story
+
+We encourage you to follow up with a specific browser vendor if you'd like more information about their plans for AppCache removal.
+
+### Firefox on all platforms
+
+Firefox [deprecated](https://www.fxsitecompat.dev/en-CA/docs/2015/application-cache-api-has-been-deprecated/) AppCache in release 44 (September 2015), and has [removed](https://www.fxsitecompat.dev/en-CA/docs/2019/application-cache-storage-has-been-removed-in-nightly-and-early-beta/) support for it in its Beta and Nightly builds as of September 2019.
+
+### Safari on iOS and macOS
+
+Safari [deprecated](https://bugs.webkit.org/show_bug.cgi?id=181764) AppCache in early 2018.
+
+### Chrome on iOS
+
+Chrome for iOS is a special case, as it uses a different browser engine than Chrome on other platforms: the [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview). Service workers are not currently supported on Chrome for iOS, and Chrome's AppCache removal announcement does not cover the [availability of AppCache on Chrome for iOS](https://webkit.org/status/#specification-application-cache). Please keep this in mind if you know that your web app has a significant Chrome for iOS audience.
```

Comment:
Do all WKWebView browsers lack service worker support? Maybe you should expand the iOS advice to cover this.

Reply:
I'll reword "Service workers are not currently supported on Chrome for iOS" to "Service workers are not currently supported in iOS apps using WKWebView."